### PR TITLE
Clean up verification tools

### PR DESCRIPTION
**Pull request by chuckha**
Signed-off-by: Chuck Ha <chuckh@vmware.com>
**What this PR does / why we need it**:
This PR cleans up the verify scripts to make the output tell us what failed in a clean way.

```release-note
NONE
```

### CODE REVIEW

File: hack/verify-all.sh [hunk 1]
```diff
@@ -13,101 +13,91 @@
...
 fi
 
 if [[ "${VERIFY_BUILD:-true}" == "true" ]]; then
   echo "[*] Verifying build..."
-  hack/verify-build.sh
-  failure $? "verify-build.sh"
-  cd "${REPO_PATH}"
-fi
-
-if [[ "${VERIFY_DOCKER_BUILD:-true}" == "true" ]]; then
```
Comment (ashish-amarnath) [inline hack/verify-all.sh]: Is this removal intentional?

File: hack/verify-all.sh [hunk 2]
```diff
@@ -13,101 +13,91 @@
...
 fi
 
 if [[ "${VERIFY_GOFMT:-true}" == "true" ]]; then
   echo "[*] Verifying gofmt..."
-  hack/verify-gofmt.sh
-  failure $? "verify-gofmt.sh"
-  cd "${REPO_PATH}"
-fi
-
-if [[ "${VERIFY_GOIMPORTS:-true}" == "true" ]]; then
```
Comment (ashish-amarnath) [inline hack/verify-all.sh]: Is this removal intentional?

Comment (chuckha) [reply hack/verify-all.sh]: no!!
good catch

File: hack/verify-all.sh (hunk 1 above)
Comment (chuckha) [reply hack/verify-all.sh]: again, no!

**Comment by chuckha** [comment]
fixed! ptal @ashish-amarnath

File: hack/verify-all.sh [hunk 3]
```diff
@@ -13,101 +13,105 @@
...
-set -o errexit
 set -o nounset
 set -o pipefail
 
 # shellcheck source=/dev/null
 source "$(dirname "$0")/utils.sh"
 
 # set REPO_PATH
-readonly REPO_PATH=$(get_root_path)
+REPO_PATH=$(get_root_path)
```
Comment (ashish-amarnath) [inline hack/verify-all.sh]:
```suggestion
readonly REPO_PATH=$(get_root_path)
```

File: hack/verify-all.sh [hunk 4]
```diff
@@ -117,6 +121,11 @@ if [[ "${res}" = 0 ]]; then
   echo "All verify checks passed, congrats!"
 else
   echo ""
```
Comment (ashish-amarnath) [inline hack/verify-all.sh]: can this be removed?

Comment (ashish-amarnath) [inline hack/verify-all.sh]:
```suggestion
  
```

Comment (chuckha) [reply hack/verify-all.sh]: this is for aligning the output. It's nicer with it in.

Comment (ashish-amarnath) [reply hack/verify-all.sh]: 👍
